drgn/helpers/linux/mm.py

```diff
@@ -1320,9 +1320,9 @@ def access_remote_vm(mm: Object, address: IntegerLike, size: IntegerLike) -> byt
     return _linux_helper_read_vm(mm.prog_, mm.pgd, address, size)
 
 
-def cmdline(task: Object) -> List[bytes]:
+def cmdline(task: Object) -> Optional[List[bytes]]:
     """
-    Get the list of command line arguments of a task.
+    Get the list of command line arguments of a task, or None for kernel tasks.
```

Comment:
One more typographical thing:

```suggestion
    Get the list of command line arguments of a task, or ``None`` for kernel tasks.
```